mono/metadata/method-builder-ilgen-internals.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * \file
+ * Copyright 2018 Microsoft
+ * Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+#ifndef __MONO_METHOD_BUILDER_ILGEN_INTERNALS_H__
+#define __MONO_METHOD_BUILDER_ILGEN_INTERNALS_H__
+
+#include "config.h"
+#include <mono/metadata/class.h>
+#include <mono/metadata/object-internals.h>
+#include <mono/metadata/class-internals.h>
+#include <mono/metadata/opcodes.h>
+#include <mono/metadata/reflection.h>
+#include <mono/metadata/method-builder.h>
+
+/* ilgen version */
+struct _MonoMethodBuilder {
+	MonoMethod *method;
+	char *name;
```

Comment:
You could fix up the types to be eglib types: `char` -> `gchar`, `int` -> `gint`, `unsigned char` -> `guchar`, etc.

Reply:
good point, let me do that in a follow up commit in master.